cypress/integration/usePrefersColorScheme.spec.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="Cypress" />
+
+describe('usePrefersColorScheme hook', () => {
+  beforeEach(() => {
+    cy.visit('/iframe.html?path=/story/utility-useprefersrcolorscheme--example-usage');
+    // Setting viewport dimensions to avoid side effects
+    cy.viewport(500, 500);
+  });
+
+  it('should set dark or light', () => {
+    const DARK = '(prefers-reduced-motion: dark)';
+    const LIGHT = '(prefers-reduced-motion: light)';
+
+    if (window.matchMedia(DARK).matches) {
+      cy.contains('reduce').should('exist');
```

Comment:
These assertions aren't running for me correctly. I think we need to do this: https://www.cypress.io/blog/2019/12/13/test-your-web-app-in-dark-mode/

```js
cy.visit('/iframe.html?path=/story/utility-usepreferscolorscheme--example-usage', {
      onBeforeLoad(win) {
        cy.stub(win, 'matchMedia')
          .withArgs('(prefers-color-scheme: dark)')
          .returns({
            matches: true,
          });
      },
    });
```